### PR TITLE
fix(ais-current-refinements): support Query refinements

### PR DESCRIPTION
**Pull request by tkrugg**
image

### CODE REVIEW

File: src/current-refinements/__tests__/__snapshots__/current-refinements.spec.ts.snap [hunk 1]
```diff
@@ -279,3 +279,83 @@ exports[`CurrentRefinedValues should be hidden with \`autoHideContainer\` 1`] =
...
+              Query:
+            </span>
+            
+            <span
+              class="ais-CurrentRefinements-category"
+            >
+              <span
+                class="ais-CurrentRefinements-categoryLabel"
+              >
+                my query
```
Comment (francoischalifour) [inline src/current-refinements/__tests__/__snapshots__/current-refinements.spec.ts.snap]: The query should be wrapped with double quotes according to the [spec](https://instantsearch-css.netlify.com/widgets/current-refinements/) (cc @Haroenv).

File: src/current-refinements/current-refinements.ts [hunk 2]
```diff
@@ -89,7 +89,11 @@ export class NgAisCurrentRefinements extends BaseWidget {
         : this.state.refinements;
 
     // group refinements by category? (attributeName && type)
-    return items.reduce((res, { type, attributeName, ...refinement }) => {
+    return items.reduce((res, { type, attributeName: _attributeName, ...refinement }) => {
+      let attributeName = _attributeName;
```
Comment (francoischalifour) [inline src/current-refinements/current-refinements.ts]: I would rather go for a ternary here.

```js
const attributeName = type === 'query' ? 'query' : _attributeName,
```

**Comment by samouss** [comment]
We have a similar issue in InstantSearch (see https://github.com/algolia/instantsearch.js/issues/3205) worth fixing it directly on the InstantSearch then update the IS version inside Angular.

**Comment by francoischalifour** [comment]
@samouss I wasn't aware of the issue in InstantSearch.js. It's fixed on the `feat/3.0` branch. Indeed, it would make more sense to fix it on the InstantSearch.js `develop` branch.

**Comment by samouss** [comment]
Yep still good to fix it on InstantSearch for the users that will stay on this version.

File: src/current-refinements/__tests__/__snapshots__/current-refinements.spec.ts.snap (hunk 1 above)
Comment (Haroenv) [reply src/current-refinements/__tests__/__snapshots__/current-refinements.spec.ts.snap]: Wrapped in a q tag indeed.

**Comment by tkrugg** [comment]
closing this in favour of #3205

**Pull request closed without merging**